apps/andi/lib/andi_web/live/ingestion_live_view/extract_steps/extract_http_step_form.ex

```diff
@@ -52,7 +52,7 @@ defmodule AndiWeb.ExtractSteps.ExtractHttpStepForm do
 
                 <div class="extract-http-step-form__url">
                   <%= label(f, :url, DisplayNames.get(:url), class: "label label--required") %>
-                  <%= text_input(f, :url, [id: "step_#{@id}__url", class: "input full-width", disabled: @testing, required: true]) %>
+                  <%= url_input(f, :url, [id: "step_#{@id}__url", class: "input full-width", disabled: @testing, required: true]) %>
```

Comment:
Interesting. What does this do? The docs are very descriptive and say "Generates an url input."

Reply:
Nothing, it's just an alias for text_input. I liked that it was a little more descriptive.